MediaGalleryUi/Ui/Component/ImageUploaderStandAlone.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\MediaGalleryUi\Ui\Component;
+
+use Magento\Framework\UrlInterface;
+use Magento\Framework\View\Element\UiComponent\ContextInterface;
+use Magento\Ui\Component\Container;
+
+/**
+ * Image Uploader component
+ */
+class ImageUploaderStandAlone extends Container
+{
+    const ACCEPT_FILE_TYPES = '/(\.|\/)(gif|jpe?g|png)$/i';
+    const ALLOWED_EXTENSIONS = 'jpg jpeg png gif';
+    const MAX_FILE_SIZE = '2097152';
+
+    /**
+     * @var UrlInterface
+     */
+    private $url;
+
+    /**
+     * Constructor
+     *
+     * @param ContextInterface $context
+     * @param UrlInterface $url
+     * @param array $components
+     * @param array $data
+     */
+    public function __construct(
+        ContextInterface $context,
+        UrlInterface $url,
+        array $components = [],
+        array $data = []
+    ) {
+        parent::__construct($context, $components, $data);
+        $this->url = $url;
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function prepare(): void
+    {
+        parent::prepare();
+        $this->setData(
+            'config',
+            array_replace_recursive(
+                (array) $this->getData('config'),
```

Comment:
Hi @Nazar65, just to re-check, here you replace the values from config in layout with default values in the array? Shouldn't it be the opposite?

Reply:
nope, in this case we need replace it in array from backend